### PR TITLE
Beacon copy changes

### LINKED ISSUE #233: Tweak mobile dropdown
Mobile dropdown items should have a margin-left of 15px, otherwise they're out of alignment and hug the screen, a la: 
screen shot 2015-08-18 at 11 47 56 am

### LINKED ISSUE #305: Some feedback on Beacon welcome email from Rebecca Harris
Welcome email
- Should be more like a welcome packet
  - Make sure to share what _exactly_ beacon does
- Also point people to BidNet directly (register to be on there for free)
- Maybe also point to other known places

Front page
- Should have additional contact information (phone numbers?)
- How do people know that this is legitimate? Would it be possible to get a city seal/seal of the Department of I&P to put in the footer or somewhere else?
- There's no real way to know what Beacon ~is~ actually
- Nothing about the professional services on the front page
- The text right under the "The City of Pittsburgh could be..." isn't right. Maybe something along the lines of "The City of Pittsburgh needs to buy what you sell. Sign up to get notified about..."

About page
- A bit about CfA, a bit about our partnership with the City.

Other
- Is there any way to piggyback on the Mayor's Innovation Roadmap announcement? (September 8, should be in email, see Slack also)

### LINKED ISSUE #322: Beacon nav formatting issues for phone / tablet resolutions
Margins look funky. We should fix in the event that people at the press conference are on mobile, and to take a nice screencap for the slide. 

screen shot 2015-09-01 at 3 20 08 pm

screen shot 2015-09-01 at 3 20 52 pm

### PR DESCRIPTION
**Pull request by hamhands**
### What changed
- Footer changes, logo additions
- Changes to splash copy (splopy)
- Fixes for navbar mobile issues
- add some kind of about info or page
- prof services content
### Issues
- Fixes #305 
- Fixes #322
- Fixes #233 
### Todo
- ~~add OMB contact info~~ (asked to hold off by Slim)
### Screencap

screen shot 2015-09-01 at 5 28 49 pm
screen shot 2015-09-01 at 5 28 40 pm

### CODE REVIEW

**Comment by msknee** [comment]
:+1:

**Comment by msknee** [comment]
@bsmithgall @hamhands can we merge this as is so I can make a licecap for the slide?

**Comment by hamhands** [comment]
@bsmithgall @shellicious I think this is ready to merge whenever yinz is ready.

**Comment by msknee** [comment]
oooo yayyyyy can you put a screencap of the FAQ copy on here?

**Comment by msknee** [comment]
:dizzy:

**Comment by hamhands** [comment]
NOPE.

jk
screen shot 2015-09-02 at 11 55 58 am

File: purchasing/templates/opportunities/front/splash.html [hunk 1]
```diff
@@ -84,8 +86,13 @@
         </div>
 
         <div class="faq-question row">
+          <h3><strong>Who made this?</strong></h3>
+          <p>This tool was made as part of a year-long partnership between the City of Pittsburgh and a non-profit called <a href="http://www.codeforamerica.org">Code for America</a>, leveraging modern technology and user-centered design to make improvements to how products and services are purchased with City government. <a href="http://www.codeforamerica.org/governments/pittsburgh/">Click here</a> to learn more about this project and the team behind it.</p>
```
Comment (bsmithgall) [inline purchasing/templates/opportunities/front/splash.html]: The first sentence here is a bit awkward I think.

File: purchasing/static/img/Pittsburgh_city_coat_of_arms.svg [hunk 2]
```diff
@@ -0,0 +1,561 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (bsmithgall) [inline purchasing/static/img/Pittsburgh_city_coat_of_arms.svg]: Are we using this file anywhere? If not, maybe just include the .png

Comment (hamhands) [inline purchasing/static/img/Pittsburgh_city_coat_of_arms.svg]: yes, it's in the footer

On Wed, Sep 2, 2015 at 12:28 PM, Ben Smithgall notifications@github.com
wrote:

> In purchasing/static/img/Pittsburgh_city_coat_of_arms.svg
> https://github.com/codeforamerica/pittsburgh-purchasing-suite/pull/324#discussion_r38573614
> :
> 
> > @@ -0,0 +1,561 @@
> > +<?xml version="1.0" encoding="utf-8"?>
> 
> Are we using this file anywhere? If not, maybe just include the .png
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/codeforamerica/pittsburgh-purchasing-suite/pull/324/files#r38573614
> .

## 

Patrick Hammons
@hamhandedly

**Comment by bsmithgall** [comment]
What else needs to be done. Does this resolve the two issues mentioned in the PR main text? If so, can you explicitly say that it does?

**Pull request merged**